docs/sources/style-guide/style-conventions/index.md

```diff
@@ -90,14 +90,45 @@ Use the following techniques to enhance the scannability of content.
 - Use short, bulleted lists.
 - Use headings to divide content.
 
-## Unordered lists
+## Lists
+
+### Ordered lists
```

Comment:
I don't want this comment to be a blocker, and I believe I said it somewhere else (Slack maybe?)... Adding guidance for nested lists is awesome, but personally I would rather not give people too different paths and let them decide if their list is big enough to use one path over another.

Reply:
I'd also support that behavior but the outcome of the style roundtable was to follow this style. That being said, it wasn't brought to a poll. I'll add it for a discussion in the next meeting.